Header header
uint32 node_count
TopologicalNode[] node
uint32 edge_count
TopologicalEdge[] edge

	
